### ISSUE TITLE
.org Plans: Duplicate emails for Jetpack Professional Plan Purchase

### ISSUE THREAD

**Issue by chaselivingston** [Automattic/wp-calypso#6334]
#### Steps to reproduce
1. Starting at URL: wordpress.com/plans
2. Purchase the Jetpack Professional Plan (I purchased with credits if that matters)
3. Get two notification emails that say the same thing (Set up your Premium Services)
#### Context / Source

# manual-testing #flowsharing

**Comment by chaselivingston** [comment]
cc @bubel

**Comment by roundhill** [comment]
Same thing happened to me yesterday:

<img width="275" alt="screen shot 2016-10-20 at 10 37 10 am" src="https://cloud.githubusercontent.com/assets/789137/19574058/85720ff8-96bd-11e6-83b9-b4b553658d8a.png">

**Comment by oskosk** [comment]
I've just experienced something similar. But not so much. I upgrade a Jetpack site to professional but got two emails stating another Jetpack site's professional plan was cancelled :S

* Site **115883770** was on a free Jetpack plan
* Site 111546322 was on a Jetpack Professional plan
* I went to upgrade **115883770** to a Professional plan.
* I upgraded **115883770** to Jetpack Professional plan
* I received two emails stating that site 111546322 ’s professional plan was cancelled. (identical to those mentioned by @roundhill. 
* I received a single email stating that site **115883770** was on a professional plan

Both sites are now on a Professional plan in spite of the cancellation mails
